example.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (fastify, options, next) {
   fastify.register(hmac, {
     sharedSecret: 'topSecret',
     getAlgorithm: () => 'sha512',
-    getDigest: () => 'base64'
+    getHMACEncoding: () => 'base64'
```

Comment:
Can we rename this to signatureEncoding? Keep it consistent with the existing API?

Reply:
It also can probably just be a string? not sure if we will need to determine the encoding programmatically with a function call ever.

Reply:
I renamed this to `getSignatureEncoding` but left it as a function. I'll handle the string vs. function under issue #3